wcfsetup/setup/db/install.sql

```diff
@@ -1432,6 +1432,19 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY,
+	title VARCHAR(255) NOT NULL DEFAULT '',
+	description TEXT, 
+	imageUrl TEXT, 
+	imageType ENUM('SQUARED', 'COVER', 'NOIMAGE') NOT NULL DEFAULT 'NOIMAGE',
+	imageHash VARCHAR(40) NOT NULL DEFAULT '',
```

Comment:
Do you need a dedicated `imageHash`? Can't you simply reuse the `urlHash` for the image? Especially since you only have one image per URL?

Reply:
No, not really. But then I need a separate column to save the file type.

Reply:
But actually yes: Otherwise I don't know, whether I have saved the image, or not (because the image proxy is enable previously)…